test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -47,16 +48,439 @@ class TestDupAccLogForResv(TestFunctional):
 
     def setUp(self):
         TestFunctional.setUp(self)
+        if len(self.moms) != 2:
+            self.skipTest('test requires two MoMs as input, ' +
+                          'use -p moms=<mom1>:<mom2>')
+
+    def check_log_resv_time(self, accounting_log_msg, start_tup,
+                            end_tup, check_status):
+        """
+        This is a helper function to reterive start and end time
+        of standing rervation from accounting log.
+        """
+        loglist_value = accounting_log_msg[start_tup][end_tup]
+        log_value = loglist_value.split()
+        if check_status == "start":
+            for attribute in log_value:
+                attribute_value = attribute.split("=")
+                if attribute_value[0] == 'start':
+                    actual_reservation_time = attribute_value[1]
+        elif check_status == "end":
+            for attribute in log_value:
+                attribute_value = attribute.split("=")
+                if attribute_value[0] == 'end':
+                    actual_reservation_time = attribute_value[1]
```

Comment:
I believe we can combine the if and elif block. Also, we should break out of the for loop once we have found both "start" and "end".